beacon/blockchain/types.go

```diff
@@ -49,6 +49,11 @@ type ConsensusBlock interface {
 	// GetConsensusTime returns the timestamp of current consensus request.
 	// It is used to build next payload and to validate currentpayload.
 	GetConsensusTime() math.U64
+
+	// GetConsensusSyncing signals whether consensus is working normally
+	// or is still syncing. In the former case we can skip execution payload
+	// verification on finalized blocks.
```

Comment:
```suggestion
	// GetConsensusSyncing returns true if consensus is still syncing, false if the node i
        // is at the head of the chain. If still syncing we can skip execution payload 
        // verification on already finalized blocks.
```
mainly wanted to remove the "working normally" and make it clear when true is returned.